examples/schemas/characters.ts

```diff
@@ -1,12 +1,18 @@
 import type { Schema } from '../../types/persistence_schema/mod.ts'
+import moviesSchema from './movies.ts'
 
 const charactersSchema: Schema = {
   source: 'characters',
   columns: [
     { name: 'id', type: 'uuid', autoGenerate: true, primaryKey: true },
     { name: 'name', type: 'text' },
     { name: 'age', type: 'numeric' },
-    { name: 'movie_id', type: 'uuid' },
+    {
+      name: 'movie',
+      type: 'belongs_to',
+      referenceKey: 'movie_id',
```

Comment:
参考下 https://hexdocs.pm/ecto/3.12.0/Ecto.Schema.html#belongs_to/2
这里好像应该叫 foreignKey 比较合适

```suggestion
      foreignKey: 'movie_id',
```